Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md

```diff
@@ -0,0 +1,301 @@
+# Hermetic Grimoire of Aine fila Saoirse
+
+This is an hermetic grimoire to store Aines spells, there are section for every technique-form combination and spells are ordered by their level.
+
+# Animal
+
+## Creo Animal
+
+## Intellego Animal
+
+## Muto Animal
+
+## Perdo Animal
+
+## Rego Animal
+
+# Aquam
+
+## Creo Aquam
+
+**Flamable lube never hurt nobody**  
+CrAq 10  
+**R:** Voice, **D:** Dia, **T:** Ind  
+Caster flics their finger and covers an individual in highly flamable oil. If it gets ignited it does +12 damage the first round, +6 damage the second round, +0 damage the third round.    
+**Mastery:** Multi    
```

Comment:
Mastery are properties of how a magi knows a spell, not of the spell itself, so it doesn't belong in a file distinct from the character.

Also, I think mastery needs level in the ability, xp, and then the list of masteries options (Multi-cast, imperturbable casting etc)